Mu2eG4/fcl/prolog.fcl

```diff
@@ -185,6 +185,7 @@ Mu2eG4MT: {
         g4run : {
             @table::mu2eg4runDefaultSingleStage
             module_type : "Mu2eG4MT"
+            initialSeed : 8 
         }
     }
 }
```

Comment:
We can use the Seed Service, but we cannot use the Random Number Engine Service in Mu2eG4MT_module.cc because it is a Shared Module.  I have altered the code to use the Seed Service.  The MTRunManager now gets a seed from the Seed Service.  It uses this seed in its MixMax random number generator to generate seeds for the worker random number generators.

Reply:
I don't see the change.  Did you push it? The last commit I see is 3 days ago.

Reply:
I have just pushed the change.